### ISSUE TITLE
Label tags on settings page for accessibility

### ISSUE THREAD

**Issue by steverep** [awesomemotive/google-analytics-for-wordpress#313]
I'm using the latest versions of Wordpress and the GA plugin.  As far as I can tell, many form fields (check boxes, combo-boxes, etc.) do not use the `<label>` tag to improve accessibility for screen readers.  Please do a review of this and properly label all form fields on the Settings tabs.  Thanks!

P.S. I'm a little surprised at this, because the MANY fields added by the Yoast SEO plugin have the <label> tag and are quite accessible.

**Comment by tacoverdo** [comment]
Forwarded to the development team. This is to be fixed as soon as possible.

**Comment by steverep** [comment]
Thanks! Also, I noticed after I first submitted this that the few combo-boxes and other fields on the Dashboard tab also have this issue.

**Comment by steverep** [comment]
I made the mistake of thinking this was a quick fix and tried figuring it out myself, and then I dug deeper into the code and how the fields are eventually output in a fairly complex class structure (which surpasses my PHP expertise level at the moment).  I'm wondering, though, if the fix is simply to modify the admin/class-settings-registrar.php file to include \<label for="id"\>...\</label\> around the text for most fields, although I couldn't figure out where the id should come from.  It is eventually set to "ga_form_" . $args['key'] . '"' in most cases that I saw.

Anyway, I fear I'm way off track and the fix to properly do this requires more work as the current code kind of paints accessible forms into a corner.  Let me know.
